app/controllers/cypress/user_creator_playwright_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Cypress
```

Comment:
This is called `Cypress` on purpose since it allowed us to easily extend the existing Cypress routes. Once we migrated everything to Playwright, we can easily rename this.